sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/models/ResponseBodyMode.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.http.models;
+
+import java.io.InputStream;
+
+/**
+ * Enum that defines how to handle the body of an HTTP response.
+ */
+public enum ResponseBodyMode {
```

Comment:
Did we finalize this name or should we consider naming this `ResponseHandlingMode`?

Reply:
I prefer `ResponseBodyHandlingMode`, but I think `ResponseHandlingMode` is slightly better than `ResponseBodyMode`.